src/content/tooling/playground.md

```diff
@@ -87,6 +87,7 @@ While the preview tool is powerful in development, the tool must run as a mock p
 The key differences of the previewer versus a live Workers script are:
 
 - Subrequests ( i.e. fetches inside your Workers script) that call the same hostname as the original request will run in the previewer as an external request (i.e. goes back to the front line of Cloudflare), as those subrequests in production will go directly to the origin.
+- Subrequest fetches to IPv6-only hosts are not supported.
```

Comment:
```suggestion
-  `fetch` can only connect to IPv4 addresses in the playground thus, IPv6-only hosts are not supported.
```

Reply:
This was worded a little funny it made me think we only supported IPv6 when I read it